### PR TITLE
CM-704 - Updates to copy and icons in the Top Up views 

### PR DESCRIPTION
**Pull request by alejoloaiza**
# Summary
- Fix: Top up screen needs to be updated as per figma shared, see image
image (12)
This is a screen taken from the widget running locally after the changes were implemented.
Screenshot 2024-06-05 at 1 18 32 PM

# Detail and impact of the change

## Security

# Anything else worth calling out?

### CODE REVIEW

**Review by jwhardwick — APPROVED** [review]

**Review by mimi-imtbl — APPROVED** [review]

File: packages/checkout/widgets-lib/src/locales/en.json [hunk 1]
```diff
@@ -231,30 +231,36 @@
     },
     "TOP_UP_VIEW": {
       "header": {
-        "title": "How would you like to add coins?"
+        "title": "How would you like to add tokens?"
```
Comment (jhesgodi) [inline packages/checkout/widgets-lib/src/locales/en.json:234]: Must remain as `coins` unless updating this across all relevant screens

File: packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx [hunk 2]
```diff
@@ -57,6 +57,15 @@ interface TopUpViewProps {
   subheading?: [key: string, options?: $Dictionary];
 }
 
+type TopUpFeatures = Partial<TopUpMenuItemProps> & {
```
Comment (jhesgodi) [inline packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx:60]: noice

File: packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx [hunk 3]
```diff
@@ -352,7 +373,9 @@ export function TopUpView({
               <TopUpMenuItem
                 key={t(`${element.textConfigKey}.heading`).toLowerCase()}
                 testId={element.testId}
-                icon={element.icon as 'Wallet' | 'Coins' | 'Minting'}
+                icon={element.icon as 'Swap' | 'BankCard' | 'Minting' | 'ArrowForward'}
```
Comment (jhesgodi) [inline packages/checkout/widgets-lib/src/views/top-up/TopUpView.tsx:376]:
```suggestion
icon={element.icon}
```

File: packages/checkout/widgets-lib/src/locales/en.json (hunk 1 above)
Comment (alejoloaiza) [reply packages/checkout/widgets-lib/src/locales/en.json:234]: The designs shared by UI/UX said tokens in there.

**Review by jhesgodi — APPROVED** [review]
Awesome work @alejoloaiza

**Review by mimi-imtbl — APPROVED** [review]

**Pull request merged**